strongbox-distribution/src/main/resources/META-INF/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
In regards to this module:
* In its root it contains a `LICENSE` which you can remove.
* Could you please check where the `LICENSE` from `src/main/resources` ends up in the `strongbox-distribution` archives?

Reply:
If we need to add `LICENSE` in archive distribution, we can add otherwise we need to remove it from `src/main/resources`.

Reply:
@BhuwanUpadhyay : 

You will have to update `strongbox-distribution/src/main/assembly/distribution.xml` to include the `LICENSE` and place it inside the `strongbox-distribution-${version}/strongbox-${version}` directory.

Reply:
@carlspring I made a change for it and verified.

Reply:
So, what's the full path to the license file in the archive files now?

Reply:
@carlspring It is under `/strongbox-distribution-<version>/strongbox-<version>/LICENSE`.

Reply:
Excellent, thank you! ;)